assembl/views/__init__.py

```diff
@@ -226,10 +282,25 @@ def process_export_list(ls):
 
     analytics_url = config.get('web_analytics_piwik_url', None)
 
+    get_route = create_get_route(request, discussion)
+    providers = get_provider_data(get_route)
+
+    errors = request.session.pop_flash()
+    if kwargs.get('error', None):
+        errors.append(kwargs['error'])
+    if errors:
+        kwargs['error'] = '<br />'.join(errors)
+    messages = request.session.pop_flash('message')
+    if messages:
+        kwargs['messages'] = '<br />'.join(messages)
+
+    admin_email = config.get('assembl.admin_email') or "assembl.admin@bluenove.com"
```

Comment:
I think it's overkill to not start Assembl if the admin_email isn't available. But, I'm not clear on downstream implications. I am okay with an exception raised here if admin email doesn't exist.